### PR TITLE
Add options for [--image] command.

### PR DESCRIPTION
**Pull request by 2016321**
If you want to use this tool in large project, you will get a much too crowded image. I have added ranksep & nodesep options for graphviz.

### CODE REVIEW

File: lib/pod/command/dependencies.rb [hunk 1]
```diff
@@ -32,6 +34,16 @@ def initialize(argv)
         @produce_graphviz_output = argv.flag?('graphviz', false)
         @produce_image_output = argv.flag?('image', false)
         @use_podfile_targets = argv.flag?('use-podfile-targets', false)
+        @ranksep = "0.75"
+        @nodesep = "0.25"
+        ranksep = argv.all_options('ranksep').at(0)
```
Comment (segiddins) [inline lib/pod/command/dependencies.rb]: can this use `argv.option('rankest', '0.25')` ?

Comment (2016321) [reply lib/pod/command/dependencies.rb]: Yep, it's briefer

**Comment by dnkoutso** [comment]
Still want to merge this?

**Comment by 2016321** [comment]
@dnkoutso Yep, i have changed yet.  When i use default parameter, i get a messy image like this.

image
But when i use `--image --ranksep=5 --nodesep=.5`, the result like this

image

I know that our project must be more graceful. But i dont'n want to get a useless image when i use this plugin.

**Comment by segiddins** [comment]
Thanks!

**Pull request merged**